internal/task/task/common/client_status.go

```diff
@@ -81,11 +93,73 @@ func (s *step2FormatClientStatus) Execute(ctx *context.Context) error {
 		Kind:        client.Kind,
 		ContainerId: s.containerId,
 		Status:      status,
+		Address:     address,
 		AuxInfo:     client.AuxInfo,
 	})
 	return nil
 }
 
+func (s *step2GetAddress) Execute(ctx *context.Context) error {
+	cmd := ctx.Module().DockerCli().TopContainer(s.containerId)
+	out, err := cmd.Execute(s.execOptions)
+	if err != nil {
+		return nil
```

Comment:
return err?

Reply:
thx, I haven't tested the situation that top command goes wrong, but why it throw error